package/MDAnalysis/coordinates/NAMDBIN.py

```diff
@@ -113,15 +113,14 @@ def __init__(self, filename, n_atoms=None, **kwargs):
         """
         self.filename = util.filename(filename)
 
-    def write(self, obj):
+    def write_next_timestep(self, obj):
         """Write obj at current trajectory frame to file.
 
         Parameters
         ----------
-        obj : :class:`~MDAnalysis.core.groups.AtomGroup` or :class:`~MDAnalysis.core.universe.Universe` or a :class:`Timestep` 
+        obj : :class:`~MDAnalysis.core.groups.AtomGroup` or :class:`~MDAnalysis.core.universe.Universe`
               write coordinate information associate with `obj`
         """
-

```

Comment:
need a deprecation warning here when using a `ts`?

`write_next_timestep()` is this the name we're using even though we don't really want to use `timestep`?